src/pages/_document.tsx

```diff
@@ -27,7 +28,7 @@ export default class MyDocument extends Document {
       ? getInstanceDataByLang(this.props.__NEXT_DATA__.locale as Instance)
       : undefined
     return (
-      <Html className="print:serlo-print-style">
+      <Html className="print:serlo-print-style bg-brand-100">
```

Comment:
@elbotho can you add a quick note about what this change does?

Reply:
If you "overscroll" (only possible on some devices) you see the background color of the html element. It's just a tiny mostly visual thing.

Reply:
Thanks, makes sense. If you don't mind, you could add your explanation as a comment, so that nobody "refactors" it away.